packages/table/src/TableV10/Table.tsx

```diff
@@ -108,7 +108,8 @@ export interface TableRowInterface<Shape = {}> {
  * @deprecated
  * @noDocgen
  */
-export interface TableProps<Shape> extends HTMLElementProps<'table'> {
+export interface TableProps<Shape>
```

Comment:
Does anyone know why and when this was changed?

Reply:
@spark33

Reply:
Changed to `extends Omit<HTMLElementProps<'table'>, 'children' | 'columns'>` since we're re-defining `children` & `columns`